tests/functional/lib/helpers.js

```diff
@@ -1004,6 +1004,26 @@ define([
     };
   }
 
+  function testAttribute (elementSelector, attributeName, assertion, value) {
```

Comment:
I like these new functions. Can you add some jsdocs? I just realized I forgot to add them myself to a couple of new functions I added. :( I'll open another PR to document the functions I've just added.

Do you think we should use the `testElement*` convention since these tests relate to elements?